EntAppProject/src/main/resources/templates/JobBookCSS.css

```diff
@@ -1,10 +1,10 @@
 h1	{

```

Comment:
I am not going to change this, but I do not think doing element level styles like this is a good idea. Maybe if you want all text to be green, sure? But, I do not think the text-align: center is what you want. This means that ANYTIME you create an h1, it is going to have these styles applied